src/components/Checklist.tsx

```diff
@@ -1,16 +1,32 @@
-import { FunctionComponent } from "react";
+import { FunctionComponent, useState, useEffect } from "react";
 import { ChecklistItem } from "./ChecklistItem";
-import { ListofItems } from "../App";
+import { Item, ListofItems } from "../App";
 
 type Props = { items: ListofItems };
 
 export const Checklist: FunctionComponent<Props> = ({ items }) => {
-  const taskList = items.map((task) => (
+  const [data, setData] = useState(items);
+  const [newItemName, setNewItemName] = useState("");
+
+  useEffect(() => {
+    console.log(data);
+  }, [data]);
+

```

Comment:
I think we can remove the whole useEffect since it is only logging (unless it would be useful for debugging later)